insights/combiners/tests/test_grub_conf.py

```diff
@@ -93,6 +93,117 @@
 }
 """.strip()  # noqa
 
+GRUB2_TEMPLATE_BLSCFG = """
+#
+# DO NOT EDIT THIS FILE
+#
+# It is automatically generated by grub2-mkconfig using templates
+# from /etc/grub.d and settings from /etc/default/grub
+#
+
+### BEGIN /etc/grub.d/00_header ###
+set pager=1
+
+terminal_output console
+### END /etc/grub.d/00_header ###
+
+### BEGIN /etc/grub.d/00_tuned ###
+set tuned_params=""
+### END /etc/grub.d/00_tuned ###
+
+### BEGIN /etc/grub.d/01_users ###
+
+### END /etc/grub.d/01_users ###
+insmod blscfg
+blscfg
+if [ -s $prefix/grubenv ]; then
+  load_env
+fi
+### BEGIN /etc/grub.d/10_linux ###
+menuentry 'Red Hat Enterprise Linux Server (4.18.0-240.el8.x86_64) 8.3 (Maipo)' --class red --class gnu-linux --class gnu --class os --unrestricted $menuentry_id_option 'gnulinux-3.10.0-327.el7.x86_64-advanced-4f80b3d4-90ba-4545-869c-febdecc586ce' {
```

Comment:
If blscfg is enabled these menu entries shouldn't exist if the file was outputted with grub2-mkconfig. Just figured it would save some lines of code removing them since they're not needed based on the name of the string.

Reply:
@ryan-blakley - Yes, removed the menu entries